modules/odc-connecting-components.adoc

```diff
@@ -82,8 +88,10 @@ spec:
  registryNamespace: pmacik
 ----
 .. Click *Create* to create the `OperatorSource` in your cluster.
-. Install the *PostgreSQL Database* Operator:
-.. In the *Administrator* perspective of the console, navigate to the *Operators -> OperatorHub*.
++
+This adds the PostgreSQL Database Operator, provided by Red Hat, to the *OperatorHub*.
```

Comment:
You've already said in the first sentence of the step that creating the OperatorSource "adds the PostgresSQL operator provided by Red Hat to the OperatorHub". Is this necessary to repeat again here?